### PR TITLE
refactor: use hugo to build website

### PR DESCRIPTION
**Pull request by jiacai2050**
Preview 
image

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,10 @@
 
 # vscode files
 .vscode
+
+/resources
+*lock
+/public
+/node_modules
+package.json
+package-lock.json
```
Comment (chunshao90) [inline .gitignore:15]: Add a blank line.

File: hugo.toml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+[params.links]
+[[params.links.user]]
+	name = "GitHub Discussions"
+	url = "https://github.com/apache/horaedb/discussions"
+	icon = "fa-solid fa-circle-question"
+    desc = "Discussion and help from your fellow users"
+[[params.links.user]]
+	name ="Weixin"
+	url = "https://example.org/"
```
Comment (chunshao90) [inline hugo.toml:56]: Use HoraeDB's Weixin Official Accounts.

Comment (jiacai2050) [reply hugo.toml]: Fixed.

**Review by chunshao90 — APPROVED** [review]
LGTM

**Pull request merged**